Sources/HaCWebsiteLib/Controllers/LandingUpdateFeedController.swift

```diff
@@ -8,10 +8,17 @@ import SwiftyJSON
 
 struct LandingUpdateFeedController {
 
-  static let eventPostCards : [PostCard] = EventServer.getEvents().flatMap{ event in 
+  static var eventPostCards : [PostCard] = EventServer.getCurrentEvents().flatMap{ event in 
```

Comment:
Again, caching might be unnecessary here:

```
static var eventPostCards: [PostCard] {
  EventServer.getCurrentEvents().flatMap{ event in 
    event.postCardRepresentation
  }
}
```
Would be a nice alternative and then ditch the `update` entirely. We can always optimise this later if need be but even then it might be cleaner to have some kind of subscriber thing rather than occasional manual polling.